include/gz/sim/Util.hh

```diff
@@ -263,22 +263,22 @@ namespace ignition
     /// \return The entity's latitude (deg), longitude (deg) and elevation (m).
     /// If the entity doesn't have a pose, or the world's spherical coordinates
     /// haven't been defined, this will return nullopt.
-    std::optional<math::Vector3d> IGNITION_GAZEBO_VISIBLE sphericalCoordinates(
+    std::optional<math::Vector3d> GZ_GAZEBO_VISIBLE sphericalCoordinates(
         Entity _entity, const EntityComponentManager &_ecm);
 
     /// \brief Environment variable holding resource paths.
-    const std::string kResourcePathEnv{"IGN_GAZEBO_RESOURCE_PATH"};
+    const std::string kResourcePathEnv{"GZ_SIM_RESOURCE_PATH"};
 
     /// \brief Environment variable used by SDFormat to find URIs inside
     /// `<include>`
     const std::string kSdfPathEnv{"SDF_PATH"};
 
     /// \brief Environment variable holding server config paths.
-    const std::string kServerConfigPathEnv{"IGN_GAZEBO_SERVER_CONFIG_PATH"};
+    const std::string kServerConfigPathEnv{"GZ_SIM_SERVER_CONFIG_PATH"};
 
     /// \brief Environment variable holding paths to custom rendering engine
     /// plugins.
-    const std::string kRenderPluginPathEnv{"IGN_GAZEBO_RENDER_ENGINE_PATH"};
+    const std::string kRenderPluginPathEnv{"GZ_SIM_RENDER_ENGINE_PATH"};
```

Comment:
All these env vars need to be deprecated. This could be done in a separate PR since this one is so large already

Reply:
I took your advice and reverted the env vars for `gz-sim` specifically.
https://github.com/gazebosim/gz-sim/pull/1496/commits/39c3200f39e73f66d4e2d23d1753e1531012e1b2